requirements/base.txt

```diff
@@ -1,21 +1,20 @@
 pytz==2022.1  # https://github.com/stub42/pytz
-django-netfields==1.2.2 # https://pypi.org/project/django-netfields/
+django-netfields # https://pypi.org/project/django-netfields/
 python-slugify==6.1.2  # https://github.com/un33k/python-slugify
-Pillow==9.1.1  # https://github.com/python-pillow/Pillow
 argon2-cffi==21.3.0  # https://github.com/hynek/argon2_cffi
 whitenoise==6.2.0  # https://github.com/evansd/whitenoise
 celery==5.2.7  # pyup: < 6.0  # https://github.com/celery/celery
-django-celery-beat==2.3.0  # https://github.com/celery/django-celery-beat
+django-celery-beat  # https://github.com/celery/django-celery-beat
 flower==1.0.0  # https://github.com/mher/flower
 uvicorn[standard]==0.17.6  # https://github.com/encode/uvicorn
-asgiref~=3.5.0  # https://github.com/django/asgiref
+# asgiref~=3.5.0  # https://github.com/django/asgiref
```

Comment:
Do we not need asgiref anymore?

Reply:
Oops, good catch. The problem is that I think ~80% of these are included by higher level dependencies, so it's getting installed either way, and specifying it here is just silly, imo (and makes bumping deps HELL)

Lemme look to see what is actually installing this though.

Reply:
Yep, it's provided by a few packages, chiefly, django itself. 

```
root@5f5f4364e3f8:/app# pipdeptree -r -p asgiref
asgiref==3.8.1
├── channels==3.0.4 [requires: asgiref>=3.3.1,<4]
│   └── channels-redis==3.4.0 [requires: channels<4]
├── channels-redis==3.4.0 [requires: asgiref>=3.2.10,<4]
├── daphne==3.0.2 [requires: asgiref>=3.2.10,<4]
│   └── channels==3.0.4 [requires: daphne>=3.0,<4]
│       └── channels-redis==3.4.0 [requires: channels<4]
├── Django==4.2.16 [requires: asgiref>=3.6.0,<4]
│   ├── channels==3.0.4 [requires: Django>=2.2]
│   │   └── channels-redis==3.4.0 [requires: channels<4]
│   ├── crispy-bootstrap5==0.6 [requires: Django>=2.2]
│   ├── django-allauth==0.51.0 [requires: Django>=2.0]
│   ├── django-celery-beat==2.7.0 [requires: Django>=2.2,<5.2]
│   ├── django-cors-headers==3.13.0 [requires: Django>=3.2]
│   ├── django-debug-toolbar==3.4.0 [requires: Django>=3.2]
│   ├── django-extensions==3.1.5 [requires: Django>=2.2]
│   ├── django-grip==3.4.0 [requires: Django>=1.9]
│   │   └── django-eventstream==4.5.1 [requires: django-grip>=3.0,<4]
│   ├── django-model-utils==4.2.0 [requires: Django>=2.0.1]
│   ├── django-netfields==1.3.2 [requires: Django>=1.8]
│   ├── django-redis==5.3.0 [requires: Django>=3.2]
│   ├── django-stubs==1.11.0 [requires: Django]
│   ├── django-stubs-ext==5.1.1 [requires: Django]
│   │   └── django-stubs==1.11.0 [requires: django-stubs-ext>=0.4.0]
│   ├── django-timezone-field==7.0 [requires: Django>=3.2,<6.0]
│   │   └── django-celery-beat==2.7.0 [requires: django-timezone-field>=5.0]
│   ├── djangorestframework==3.15.2 [requires: Django>=4.2]
│   │   └── drf-spectacular==0.27.2 [requires: djangorestframework>=3.10.3]
│   ├── drf-spectacular==0.27.2 [requires: Django>=2.2]
│   └── mozilla-django-oidc==2.0.0 [requires: Django>=2.2]
└── uvicorn==0.17.6 [requires: asgiref>=3.4.0]
```

Reply:
Ok yeah. I am fine working towards removing the ones that get installed via another package automatically as I think that also helps us avoid situations where we say bump django, but still have an old asgiref and that conflicts.